components/Chat/PlaceholderField.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { StyleSheet, View, ViewStyle, TextInput } from 'react-native';
+import { AppTheme } from '../../config/DefaultConfig';
+import useTheme from "../../hooks/useTheme";
+import { isIOS } from "../../utils";
+
+interface Props {
+    firstPlaceholder: string;
+    secondPlaceholder: string;
+};
+
+const PlaceholderField: React.FunctionComponent<Props> = ({
+    firstPlaceholder,
+    secondPlaceholder
+}: Props) => {
+    const theme: AppTheme = useTheme();
+
+    return (
+        <View>
+            <View style={[style.containerNew, {borderColor: theme.lightBottomColor}]}>
+                <TextInput
+                    placeholder={firstPlaceholder}
+                    placeholderTextColor={theme.lightTextColor}
+                    style={{ color: theme.textColor }}
+                />         
```

Comment:
I think this should be created as a seprate component

Reply:
I have made a separate component